.github/workflows/build.yml

```diff
@@ -65,8 +65,15 @@ jobs:
           name: packages-${{ matrix.os_name }}
           path: ./artifacts/packages
 
+      - name: Authenticate with github packages
+        uses: actions/setup-dotnet@v1
+        with:
+          source-url: https://nuget.pkg.github.com/justeat/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Probably want this step to have the same restrictions as the push itself, otherwise it'll try and run and authenticate with our feed in PRs when there's no token.

Reply:
done